featurebyte/api/target.py

```diff
@@ -67,38 +100,85 @@ def horizon(self) -> Optional[str]:
         except RecordRetrievalException:
             return self.internal_horizon
 
-    @classmethod
+    @property
+    def node(self) -> Node:
+        """
+        Representative of the current object in the graph
+
+        Returns
+        -------
+        Node
+        """
+        try:
+            node_to_return = self.cached_model.node
+        except RecordRetrievalException:
+            node_name = self.internal_node_name
+            assert node_name is not None
+            node_to_return = self.graph.get_node_by_name(node_name)
+        return node_to_return
```

Comment:
Curious where this node is used? 
```
node_to_return = self.cached_model.node   <---- this is the node of the pruned graph (stored at the persistent)
```
If it is desired to access the node on the global graph, should use something like `global_graph.get_node_by_name(sef.node_name)`.

Reply:
ah think this was sth that i added when tinkering around with a previous version, doesn't seem used now so i removed it!